tests/unit/src/test/scala/tests/StatusBarSuite.scala

```diff
@@ -3,27 +3,28 @@ package tests
 import scala.concurrent.Promise
 import scala.meta.internal.io.PathIO
 import scala.meta.internal.metals.Buffers
-import scala.meta.internal.metals.Icons
 import scala.meta.internal.metals.ProgressTicks
 import scala.meta.internal.metals.StatusBar
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.meta.internal.metals.{
   ClientExperimentalCapabilities,
   InitializationOptions
 }
+import scala.meta.internal.metals.ClientConfiguration
+import scala.meta.internal.metals.MetalsServerConfig
 
 class StatusBarSuite extends BaseSuite {
   val time = new FakeTime
   val client = new TestingClient(PathIO.workingDirectory, Buffers())
-  import scala.meta.internal.metals.StatusBarConfig
   var status = new StatusBar(
     () => client,
     time,
     ProgressTicks.dots,
-    Icons.default,
-    StatusBarConfig.default,
-    ClientExperimentalCapabilities.Default,
-    InitializationOptions.Default
+    new ClientConfiguration(
```

Comment:
`ClientConfiguration.Default` ?

Reply:
Yea that's a good idea since we do this is a couple places. I'll go ahead and add this in.